lib/src/state_manager/simple/get_view.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/widgets.dart';
 import 'package:get/src/instance/get_instance.dart';
 
+/// GetView is a great way of quickly access your Controller
+/// without having call Get.find<Controller>() yourself.
```

Comment:
@roipeker maybe you can add how to use it, something like this.

```dart
class SomeScreen extends StatelessWidget {}

// change to

class SomeScreen extends GetView<SomeController> {}
```

Reply:
Sure!, it was a quick test i was doing with @Nipodemos with the PRs yesterday.